toolkit/Makefile

```diff
@@ -85,8 +85,9 @@ endif
 
 # External source server
 SOURCE_URL         ?= 
-PACKAGE_URL        ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/base/$(build_arch)/rpms
-PACKAGE_UPDATE_URL ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/update/$(build_arch)/rpms
+PACKAGE_URL_LIST   ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/base/$(build_arch)/rpms \
+                      https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/update/$(build_arch)/rpms \
+                      https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/preview/$(build_arch)/rpms
```

Comment:
Is this making preview repo a default alongside `base` and `update`? If I'm not mistaken, the preview should be only a possibility one has to choose to use themselves, since these packages didn't get a green light to be part of the officially supported versions, yet.

Reply:
The `PACKAGE_URL_LIST` is only used to pull down exact filenames that are in the toolchain manifests. Therefore it should only ever be pull from the preview URL if using a toolkit that references preview packages in the manifest.

Thoughts? I'm open to removing the preview repo by default if we want to guarantee the behavior you mentioned.

Reply:
For now I'd remove the preview from here. Just so one doesn't build with preview RPMs by accident. If anything, we could add a switch to enable that repo to make such builds easier and so that it's a conscious decision.